content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: blog
+title: "Spotlight on SIG Node"
+date: 2021-09-27
+slug: sig-node-spotlight-2021
+---
+
+**Author:** Dewan Ahmed, Red Hat
+
+## Introduction
+
+[SIG Node](https://github.com/kubernetes/community/tree/master/sig-node) is responsible for the components that support the controlled interactions between pods and host resources. The work of this group manages the lifecycle of pods that are scheduled to a node, and focuses on enabling a broad set of workload types, including workloads with hardware specific or performance sensitive requirements. In this blog, we have summarized our conversation with [Elana Hashman (EH)](https://twitter.com/ehashdn) & [Sergey Kanzhelev (SK)](https://twitter.com/SergeyKanzhelev), who walk us through the various aspects of being a part of the SIG and share some insights about how others can get involved.
```

Comment:
This is a better introduction, but now it's mostly a copy of the answer to the first question so it's giving away some of the post's content early.

Since this PR has been merged, but the publication date is still in the future, I suggest opening a follow up PR to rewrite this introduction.

Reply:
Good catch @nate-double-u 

I didn't notice that the same was reused in the first question. I'm working on rewriting the intro and expect to create another PR by end of the day.

Reply:
@nate-double-u https://github.com/kubernetes/website/pull/29787